script/travis-deploy.sh

```diff
@@ -8,8 +8,7 @@ if [ "$TRAVIS_BRANCH" == "master" ] &&
 
   # Setup/install
   bundle install --deployment --retry 3
```

Comment:
Left an extra `bundle install` here assuming the `deployment` and `retry` proved useful. The second run will be quick, while adding those extra params is pretty slow for this always-run-installs approach.